### DISCUSSION TITLE
挂载和彩云个人盘报错

### DISCUSSION THREAD

**Discussion by Logosww** [AlistGo/alist#1242] [Q&A]
![image](https://user-images.githubusercontent.com/87473526/174292500-b54813ec-1402-4c07-8f03-1d05c7fb9c57.png)
cookies, file_id都是根据文档正确填写的，但就是会报这个错。有谁知道该怎么解决吗？

**Comment by xhofe** [comment]
https://alist-doc.nn.ci/docs/driver/base#%E8%99%9A%E6%8B%9F%E8%B7%AF%E5%BE%84

**Reply by Logosww** [reply]
啥意思，我虚拟路径填的/啊，也没开代理啊，你的意思是要开代理吗？

**Reply by anwen-anyi** [reply]
> 啥意思，我虚拟路径填的/啊，也没开代理啊，你的意思是要开代理吗？

你可能创建了一个了用过一次 `/` 当路径了，同一个虚拟路径名称只能用一次 因为这个是唯一是所以只能创建一个~

**Comment by Logosww** [comment]
好的，我知道了，谢谢~
&nbsp;


Logos
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年6月19日(星期天) 下午3:48
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] 挂载和彩云个人盘报错 (Discussion #1242)





  
啥意思，我虚拟路径填的/啊，也没开代理啊，你的意思是要开代理吗？
  
你可能创建了一个了用过一次 / 当路径了，同一个虚拟路径名称只能用一次 因为这个是唯一是所以只能创建一个~
